app/views/warehouse_reports/health/agency_performance/_table.haml

```diff
@@ -13,38 +13,28 @@
         -# %th With SSM
         -# %th.agency-performance__without Without SSM
         %th
-          .agency-performance__category.mb-1 Payment
           With Signed Care Plan
-        %th
-          .agency-performance__category.mb-1 Quality
+        %th{data: {toggle: :tooltip, title: Health::TeamPerformance::DESCRIPTIONS[:without_initial_careplan_90_122_days]}}
```

Comment:
The next time we need to mess with this thing, it sure feels like we could move the titles into the class as well and just loop over an array here pulling out titles and descriptions.  Something like
```
- Health::TeamPerformance.sections.each do |row|
  %th= row[:title]
  %th{data: {toggle: :tooltip, title: row[:description]}}
     row[:text]
```